[DJM-637] Add step redirecting to integration Log setup (#27578)
DataDog/documentation: content/en/data_jobs/dataproc.md

1. On the **Customize cluster** page, locate the **Initialization Actions** section. Enter the path where you saved the script from the previous step.

    When your cluster is created, this initialization action installs the Datadog Agent and downloads the Java tracer on each node of the cluster.

> 1. **Optional**: Enable logs collection for your cluster by following the Dataproc integration [Log collection setup][14] steps. Application logs will be correlated to Spark job run traces.

Reviewer: ```suggestion
1. **Optional**: Enable logs collection for your cluster by following the Dataproc integration [Log collection setup][14] steps. This correlates application logs to Spark job run traces.
```
